models/peer-server.js

```diff
@@ -22,9 +26,9 @@ class PeerServer {
       // Generated at
       g: timestamp,
       // URL
-      u: process.env.BUOY_URL,
+      u: this.ws_url,
```

Comment:
oh yeah, forgot to mention this/implement eslint in this repo but I generally stick to the [airbnb style guide](https://github.com/airbnb/javascript#naming--camelCase) when it comes to variable naming conventions– that is, `someVariable` instead of `some_variable`. ima merge this and fix it in another commit, so don't worry about it 👍